verification.py

```diff
@@ -4384,6 +4384,111 @@ def test_arbitrary_model_stats(self):
         )
 
 
+class MutationRateMapTest(Test):
+    def verify_subdivided(self, ts, rate_map, discrete=False):
+        outfile = self._build_filename(None, "mutation_counts")
+        sub_pos = np.unique(
+            np.sort(
+                np.concatenate(
+                    [
+                        rate_map.position,
+                        np.linspace(
+                            rate_map.start_position, rate_map.end_position, 101
+                        ),
+                    ]
+                )
+            )
+        )
+        sub_rate = [rate_map.get_rate(p) for p in sub_pos[:-1]]
+        sub_map = msprime.RateMap(position=sub_pos, rate=sub_rate)
+        t0 = msprime.sim_mutations(
+            ts, rate=rate_map, random_seed=789, discrete_genome=discrete
```

Comment:
We shouldn't use random seeds for the verification tests - they should be robust to small random effects, and we want to do different seeds every time.